diff --git a/node_modules/@pkgr/webpack/lib/config.js b/node_modules/@pkgr/webpack/lib/config.js
index 53e4985..626acc0 100644
--- a/node_modules/@pkgr/webpack/lib/config.js
+++ b/node_modules/@pkgr/webpack/lib/config.js
@@ -149,7 +149,9 @@ export default ({ entry = 'src', outputDir = 'dist', externals, type, copies = [
             hot: true,
             port,
             allowedHosts: 'all',
-            historyApiFallback: true,
+            historyApiFallback: {
+              disableDotRule: true,
+            },
         },
         entry: {
             app: entry,
